02-oop-inheritance/keypoints/2_1-6/script.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * Created by MaximilianoDaniel on 15/09/2015.
+ * Exercises done
+ * 1- Create a Movie object
+ * 2- Instantiate some of your favorite movies and play with them in the console.
+ * 3- Add a MovieObserver class that listen for "playing" and "sttoped" events
+ * 4- Publish "playing" event on Movie.play()
+ * 5- Publish "stopped" event on Movie.stop()
+ * 6- Log to console when each event is fired
+ * */
+var inheritPrototype = function (childClass, parentClass){
```

Comment:
**Minor:** space after parenthesis (should be ``(childClass, parentClass) {``.
Also, remove line break on line 12.